data-products/pom.xml

```diff
@@ -28,6 +28,11 @@
 			<name>Jcenter Repo</name>
 			<url>https://jcenter.bintray.com/</url>
 		</repository>
+			    <repository>
+            <id>cloud-store</id>
+            <url>https://oss.sonatype.org/content/repositories/orgsunbird-1144/</url>
+            </repository>	
+
```

Comment:
@ddevadat Cloud SDK 1.4.6 is available in the maven repo. So adding this repository is not required.

Reply:
removed